app/assets/javascripts/reducers/user_training_status.js

```diff
@@ -1,6 +1,6 @@
 import { RECEIVE_USER_TRAINING_STATUS } from '../constants';
 
-const initialState = [];
+const initialState = {};
```

Comment:
The data received for this reducer is an array, so we should keep the initial state in the same form.